docs/index.md

```diff
@@ -20,14 +20,14 @@ npm install --save-dev @ui5/cli
 
 #### ⚙️ Project Setup
 
-Configure your project for use with the UI5 Tooling.  
```

Comment:
These spaces must be kept. They represent a single line break in markdown. I guess this is caused by a "whitespace trimming" IDE extension you have. Maybe you can exclude `.md` files from it?